custom/abt/reports/expressions.py

```diff
@@ -64,6 +64,15 @@ def _get_question_options(cls, item, question_path, section='data'):
         question = questions.get('/' + section + '/' + "/".join(question_path), {})
         return question.get("options", [])
 
+    @classmethod
+    def _get_form_name(cls, item):
+        form = list(filter(lambda x: x['xmlns'] == item['xmlns'], Application.get(item['app_id']).get_forms()))[0]
```

Comment:
it might be nice to break this out into a separate function and cache it on `(app_id, xmlns)` since there is no reason we need to hit the db to pull the app doc and go through all the forms for every form submission